Dockerfile

```diff
@@ -20,6 +20,7 @@ RUN GIT_VERSION=$(git describe --tags --dirty --always) && \
 FROM amazonlinux:2
```

Comment:
Are there any plans to move to https://gallery.ecr.aws/amazonlinux/amazonlinux ?